### PR TITLE
add linux instructions

### PR DESCRIPTION
**Pull request by craigmulligan**
@jviotti pull this and check if you think it works?

### CODE REVIEW

**Comment by jviotti** [comment]
Sure, checking now.

**Comment by jviotti** [comment]
Some points:
- The file names are not showing the architecture: `chmod a+x path/to/Etcher-linux-.AppImage`. It should be saying `path/to/Etcher-linux-x86.AppImage` or `path/to/Etcher-linux-x64.AppImage` depending on what the user chose.
- Let's put say `Etcher-linux-<arch>.AppImage` instead of `path/to/Etcher-linux-<arch>.AppImage`, looks much better.
- For the `sudo` part, now that we remove the path prefix, let's add a period and a slash to make the execution clearer: `$ sudo ./Etcher-linux-<arch>.AppImage`

> Given that Etcher writes to devices directly, it requires superuser privileges. We currently elevate the whole application, but for your convenience, we’re working on delaying this step until the writing starts.

This doesn't read very well, I'm sure we can improve. Thoughts?

**Comment by jviotti** [comment]
Also, can we hide the "Download" button and other download-specific stuff when we show the instructions, so it becomes harder to miss?

**Comment by craigmulligan** [comment]
Good points, I'll update

**Comment by craigmulligan** [comment]
Updated @jviotti

File: static/js/main.js [hunk 1]
```diff
@@ -31,25 +31,28 @@ var app = new Vue({
...
 // Query s3
 var bucket = new s3("https://resin-production-downloads.s3.amazonaws.com", "etcher");
-
 bucket.getLatestVersion(function(version){
+  console.log(version);
    app.version = "v" + version;
    bucket.getFiles(version, function(files){
      app.downloads = files;
      bucket.getDynamicLink(files, app.os, app.mobileLink, app.dynamicLink.eventName, function(link) {
+       console.log("ssss")
```
Comment (jviotti) [inline static/js/main.js]: I believe these debugging logs should be removed.

**Pull request merged**